src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -173,21 +172,31 @@ private void PrintStrykerASCIIName()
             Console.WriteLine();
         }
 
-        private async Task PrintStrykerVersionInformationAsync()
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Major Bug", "S3168:\"async\" methods should not return \"void\"", Justification = "This method is fire and forget. Task.Run also doesn't work in unit tests")]
+        private async void PrintStrykerVersionInformationAsync()
         {
             var assembly = Assembly.GetExecutingAssembly();
             var assemblyVersion = assembly.GetName().Version;
             var currentVersion = SemanticVersion.Parse($"{assemblyVersion.Major}.{assemblyVersion.Minor}.{assemblyVersion.Build}");
 
-            Console.WriteLine($" Version: {Output.Green(currentVersion.ToString())} (beta)");
+            Console.WriteLine($"Version: {Output.Green(currentVersion.ToString())}");
             Console.WriteLine();
 
-            var nugetInfo = await StrykerNugetFeedInfo.Create();
-            var latestVersion = nugetInfo?.LatestVersion;
+            var client = new StrykerNugetFeedClient();
 
-            if (latestVersion != null && latestVersion != currentVersion)
+            var latestVersion = await client.GetLatestVersionAsync();
+            if (latestVersion > currentVersion)
             {
-                Console.WriteLine(Output.Yellow($@" A new version of Stryker.NET ({latestVersion}) is available. Please consider upgrading using `dotnet tool update -g dotnet-stryker`"));
+                Console.WriteLine(Output.Yellow($@"A new version of Stryker.NET ({latestVersion}) is available. Please consider upgrading!"));
+                Console.WriteLine();
+            }
+
+            var previewVersion = await client.GetPreviewVersionAsync();
+            if(previewVersion > currentVersion)
```

Comment:
I think this will always print when running a preview version. But thats ok for me

Reply:
That's correct. Unfortunately AssemblyVersion does not contain preview version information..